<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tabla_libros">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>81</y>
      <width>771</width>
      <height>471</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>titulo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>paginas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ver detalles</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>borrar</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>editar</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>portada</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LISTADO USANDO UN TABLE WIDGET</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
